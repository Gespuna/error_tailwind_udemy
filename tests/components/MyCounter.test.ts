import { describe, expect, test } from "vitest";
import { mount } from "@vue/test-utils";
import MyCounter from "../../src/components/MyCounter.vue";

describe("<MyCounter/>", () => {
  test("should match snapshots", () => {
    const wrapper = mount(MyCounter, {
      props: {
        value: 5,
      },
    });
    console.log("ðŸš€ ~ test ~ wrapper:", wrapper);
    // expect(wrapper.html()).toMatchSnapshot();
  });
});
