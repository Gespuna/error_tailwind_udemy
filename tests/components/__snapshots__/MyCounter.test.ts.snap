// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`<MyCounter/> > should match snapshots 1`] = `
"<section value="5">
  <h3 class="text-4xl font-bold underline">Counter: 3</h3>
  <h3>Square: 9</h3>
  <div><button>+1</button><button>-1</button></div>
</section>"
`;
